web-common/src/features/alerts/BaseAlertForm.svelte

```diff
@@ -73,12 +73,6 @@
     if (!name) return;
     $form.name = name;
   }
-
-  $: measure = $form.measure;
```

Comment:
We should still have this i feel. Right now if you deselect a measure with criteria, the criteria form shows empty value but i think the `$form.criteria` still has the old measure value. So I can still create the alert with a measure filter not actually selected. Which leads to a query error,
```
Alert check failed: failed to resolve alert: failed to execute query: failed to compile 'having': name "total_records" in expression is not a dimension or measure available in the current context
```

Perhaps we should unset the measure name in the criteria if it is not selected in the list of measures?